omise/omise-woocommerce
# [ENGA3-709] Replace credit card form with pay.js embedded form

#### 1. Objective

Replace credit card form with pay.js embedded form

[ENGA3-709](https://opn-ooo.atlassian.net/browse/ENGA3-709)

#### 2. Description of change

- replace card form with pay.js embedded form in checkout page
- replace card form with pay.js embedded form in save cards page.
- added new page for new card form customization.
- added new inputs in credit card config 
   - Card form design theme (light, dark) 
   - Added Discover card logo in supported card icons

#### 3. Quality assurance

- Tested locally 
  - create order without saving card
  - create order with saving card option
  - add new card in saved cards page.
  - customize the card form and see the result.
 

#### 4. Impact of the change

Really important to release omise.js and pay.js first 
(and wait for the CDN cache is clear)

#### 5. Priority of change

Normal
